txhelpers/cspp.go

```diff
@@ -0,0 +1,143 @@
+package txhelpers
+
+import (
+	"github.com/decred/dcrd/dcrutil/v2"
+	"github.com/decred/dcrd/wire"
+)
+
+// must be sorted large to small
+var splitPoints = [...]dcrutil.Amount{
+	1 << 36, // 687.19476736
+	1 << 34, // 171.79869184
+	1 << 32, // 042.94967296
+	1 << 30, // 010.73741824
+	1 << 28, // 002.68435456
+	1 << 26, // 000.67108864
+	1 << 24, // 000.16777216
+	1 << 22, // 000.04194304
+	1 << 20, // 000.01048576
+	1 << 18, // 000.00262144
+}
```

Comment:
This PR only looks for splits made by the current CSPP server.  Ideally the mix check would look for any denomination in sets of 3 or more plus change outputs that when combined add up to an input amount minus some fees, but that's not so straightforward.
Also, I think that might detect regular old pool ticket split transactions as mixes too since they are structured similarly.